src/services/teams/hooks/mail-text.js

```diff
@@ -1,27 +1,23 @@
-const { warning } = require('../../../logger/index');
+const { SC_SHORT_TITLE } = require('../../../../config/globals');
 
-if (process.env.SC_SHORT_TITLE === undefined) {
-	warning('process.env.SC_SHORT_TITLE is not defined');
-}
-const cloudTitle = process.env.SC_SHORT_TITLE;
-const close = `\n\nViel Spaß und einen guten Start wünscht dir dein ${cloudTitle}-Team`;
+const close = `\n\nViel Spaß und einen guten Start wünscht dir dein ${SC_SHORT_TITLE}-Team`;
 
 const inviteWithRegistration = ({
 	invitee, inviter, teamName, shortLink,
 }) => `Hallo ${invitee}!
-\nDu wurdest von ${inviter} eingeladen, dem Team '${teamName}' der ${cloudTitle} beizutreten.
-Da du noch keinen ${cloudTitle} Account besitzt, folge bitte diesem Link, um die Registrierung abzuschließen und dem Team beizutreten: ${shortLink}${close}`;
+\nDu wurdest von ${inviter} eingeladen, dem Team '${teamName}' der ${SC_SHORT_TITLE} beizutreten.
+Da du noch keinen ${SC_SHORT_TITLE} Account besitzt, folge bitte diesem Link, um die Registrierung abzuschließen und dem Team beizutreten: ${shortLink}${close}`;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 161. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5246313)